pkg/graphql/request_test.go

```diff
@@ -76,6 +76,26 @@ func TestRequest_Print(t *testing.T) {
 	assert.Equal(t, query, bytesBuf.String())
 }
 
+func TestRequest_IsNormalized(t *testing.T) {
+	request := Request{
+		isNormalized: true,
+	}
+
+	result := request.IsNormalized()
+
+	assert.True(t, result, "Expect IsNormalized to be true")
+}
+
+func TestRequest_IsValidated(t *testing.T) {
+	request := Request{
```

Comment:
This test doesn't check any business logic. We need a test that calls `ValidateForSchema` and checks `isValidated` field is set to `true`. We already have such a test: https://github.com/TykTechnologies/graphql-go-tools/blob/92655e8c2e963252c0e5a742925d0bc0d536e763/pkg/graphql/validation_test.go#L26

We can check `isValidated` is set to `true` like the following:

```go
assert.True(t, request.isValidated) 
```